components/style/properties/properties.mako.rs

```diff
@@ -1223,8 +1223,11 @@ impl ToCss for PropertyDeclaration {
             PropertyDeclaration::CSSWideKeyword(_, keyword) => keyword.to_css(dest),
             PropertyDeclaration::WithVariables(_, ref with_variables) => {
                 // https://drafts.csswg.org/css-variables/#variables-in-shorthands
-                if with_variables.from_shorthand.is_none() {
-                    dest.write_str(&*with_variables.css)?
+                match with_variables.from_shorthand {
+                    Some(shorthand) if shorthand.flags().contains(ALIAS_PROPERTY) =>
+                        dest.write_str(&*with_variables.css)?,
```

Comment:
We are setting `ALIAS_PROPERTY` to alias shorthands that acts like their longhand value.(Maybe we can rephrase it as ALIAS_SHORTHAND or ALIAS_SHORTHAND_PROPERTY) Well, currently only alias shorthand property is -moz-transform in Servo. But other normal `-moz-` and `-webkit-` prefixed aliases do the same currently because they are the same struct and they differ in parsing. Tested with some other alias properties in gecko. It also does that.